lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -517,6 +517,17 @@ public function prepare_item_for_response( $user, $request ) {
 		}
 
 		$context = ! empty( $request['context'] ) ? $request['context'] : 'embed';
+
+		if ( 'edit' === $context ) {
+			if ( empty( $data['capabilities'] ) ) {
+				$data['capabilities'] = new stdClass();
```

Comment:
I'm not sure whether this should actually default to an empty object, or `null`. Can you take a look to see if there's prior art?